### PR TITLE
New term request PIK3R2

### LINKED ISSUE #7719: [NTR/gene] PIK3R2
Preferred gene-related syndrome label
PIK3R2-related Overgrowth spectrum

Synonyms
MONDO:0100283

Parent term (use [OLS](https://www.ebi.ac.uk/ols/ontologies/mondo), or your favorite ontology browser)
MONDO:0019716

Definition
Please write the definition in the format: Any [parent class] in which the cause of the disease is a mutation in the [gene name] gene.

Any malformations/disorders resulting from pathogenic gain-of-function variants in the PIK3R2 gene. The variants can be germline or somatic

**Definition source (Please give PubMed ID, if applicable, in format PMID:#######)
PMID: 27854409; 32856318; 22729224; 26860062

Children terms (if applicable) Should any existing terms be re-classified as children underneath this new proposed term?

MONDO:0011313

Your nano-attribution (ORCID) or URL for a working group
If you don't have an ORCID, you can sign up for one [here](https://orcid.org/)
https://clinicalgenome.org/affiliation/50020/

### PR DESCRIPTION
**Pull request by yshwetar**
Closes #7719

- Definition
- 4 PMIDs for definition
- Related synonym
- Added child as a subclass of new term

### CODE REVIEW

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -574661,6 +574663,15 @@ is_a: MONDO:0022800 {source="https://clinicalgenome.org/affiliation/40065/", sou
 property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0002-3458-4839
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7379" xsd:anyURI
 
+[Term]
+id: MONDO:1040004
+name: PIK3R2-related overgrowth spectrum
+def: "Abnormal brain overgrowth resulting from pathogenic gain-of-function variants in the PIK3R2 gene. The variants can be germline or somatic." [https://clinicalgenome.org/affiliation/50020/, https://orcid.org/0000-0002-0587-4693, PMID:22729224, PMID:26860062, PMID:27854409, PMID:32856318]
```
Comment (sabrinatoro) [inline src/ontology/mondo-edit.obo:574669]: This definition suggests that there is a disease called "Abnormal brain overgrowth". Do we have such disease? 
Could we use the genus/parent term "overgrowth syndrome" with the differentia "PIK3R2 variants"?

File: src/ontology/mondo-edit.obo [hunk 2]
```diff
@@ -574661,6 +574663,15 @@ is_a: MONDO:0022800 {source="https://clinicalgenome.org/affiliation/40065/", sou
...
+synonym: "MONDO:0100283" RELATED [https://clinicalgenome.org/affiliation/50020/, PMID:22729224, PMID:26860062, PMID:27854409]
```
Comment (sabrinatoro) [inline src/ontology/mondo-edit.obo:574670]: synonym should be a string and not a Mondo ID

**Review by sabrinatoro — CHANGES_REQUESTED** [review]
@yshwetar There are some more comments to review related to the definition and the synonym

File: src/ontology/mondo-edit.obo (hunk 1 above)
Comment (yshwetar) [reply src/ontology/mondo-edit.obo:574669]: Comment addressed.

File: src/ontology/mondo-edit.obo (hunk 2 above)
Comment (yshwetar) [reply src/ontology/mondo-edit.obo:574670]: Comment addressed.

**Review by nicolevasilevsky — APPROVED** [review]
this looks good, thanks!

File: src/ontology/mondo-edit.obo [hunk 3]
```diff
@@ -574661,6 +574663,15 @@ is_a: MONDO:0022800 {source="https://clinicalgenome.org/affiliation/40065/", sou
...
 
+[Term]
+id: MONDO:1040004
+name: PIK3R2-related overgrowth spectrum
+def: "Any overgrowth syndrome resulting from pathogenic gain-of-function variants in the PIK3R2 gene. The variants can be germline or somatic" [https://clinicalgenome.org/affiliation/50020/, https://orcid.org/0000-0002-0587-4693, PMID:22729224, PMID:26860062, PMID:27854409, PMID:32856318]
+synonym: "overgrowth syndrome and/or cerebral malformations due to abnormalities in MTOR pathway genes" RELATED [https://clinicalgenome.org/affiliation/50020/, PMID:22729224, PMID:26860062, PMID:27854409]
+is_a: MONDO:0019716 {source="PMID:22729224", source="PMID:26860062", source="PMID:27854409", source="PMID:32856318", source="https://clinicalgenome.org/affiliation/50020/"} ! overgrowth syndrome
+property_value: http://purl.org/dc/elements/1.1/date "2024-07-03T14:16:10Z" xsd:dateTime
+property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0002-0587-4693
+
```
Comment (nicolevasilevsky) [inline src/ontology/mondo-edit.obo:574674]: please add the term tracker item to link to the ticket:
https://mondo.readthedocs.io/en/latest/editors-guide/f-entities/#term-tracker-item

**Review by nicolevasilevsky — CHANGES_REQUESTED** [review]
Looks good, see inline comment

@yshwetar please assign this back to me when it is ready for re-review, thanks!

File: src/ontology/mondo-edit.obo (hunk 3 above)
Comment (yshwetar) [reply src/ontology/mondo-edit.obo:574674]: Added term tracker! Thanks for the reminder

**Review by nicolevasilevsky — APPROVED** [review]

**Pull request merged**